packages/integrations/preact/src/index.ts

```diff
@@ -3,8 +3,8 @@ import { AstroIntegration } from 'astro';
 function getRenderer() {
 	return {
 		name: '@astrojs/preact',
-		clientEntrypoint: '@astrojs/preact/client',
-		serverEntrypoint: '@astrojs/preact/server',
+		clientEntrypoint: '@astrojs/preact/client.js',
+		serverEntrypoint: '@astrojs/preact/server.js',
```

Comment:
Make sure these are updated in the `exports` map!

Reply:
```
  "exports": {
    ".": "./dist/index.js",
    "./client": "./client",
    "./client.js": "./client.js",
    "./server": "./server",
    "./server.js": "./server.js",
    "./package.json": "./package.json"
  },
```

Looks like I forgot to clean up some testing in this one. Lets remove `"./client"` and `./server` from the package.json for consistency.

Reply:
Done!